src/archive/tar.rs

```diff
@@ -12,33 +12,29 @@ use walkdir::WalkDir;
 
 use crate::{
     error::FinalError,
-    info,
     list::FileInArchive,
     utils::{self, Bytes},
-    QuestionPolicy,
 };
 
 /// Unpacks the archive given by `archive` into the folder given by `into`.
+/// Assumes that output_folder is empty
 pub fn unpack_archive(
     reader: Box<dyn Read>,
     output_folder: &Path,
-    question_policy: QuestionPolicy,
+    mut display_handle: impl Write,
 ) -> crate::Result<Vec<PathBuf>> {
+    assert!(output_folder.read_dir().expect("dir exists").count() == 0);
     let mut archive = tar::Archive::new(reader);
 
     let mut files_unpacked = vec![];
     for file in archive.entries()? {
         let mut file = file?;
 
         let file_path = output_folder.join(file.path()?);
-        if !utils::clear_path(&file_path, question_policy)? {
-            // User doesn't want to overwrite
-            continue;
-        }
-
```

Comment:
Are our overwrite questions still working?

Reply:
unpack_archive is only calledby smart_which always create a new directory

So there is no files to overwrite